src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -243,8 +243,9 @@ This does not bypass the age restriction. It just accepts it automatically."</st
     <string name="revanced_restore_old_player_layout_summary">"Returns the player layout to the old style.
 Some features may not work properly in the old player layout."</string>
     <string name="revanced_replace_player_cast_button_title">Replace cast button</string>
-    <string name="revanced_replace_player_cast_button_summary">"Replace the cast button in the player with the 'Open music' button. (Experimental)
-Limitation: If one or more music that cannot be played in the playlist, it does not work normally."</string>
+    <string name="revanced_replace_player_cast_button_summary">"Replaces the cast button in the player with the 'Open music' button. (Experimental)
+
+Info: This feature does not work normally if any songs in the playlist cannot be played."</string>
```

Comment:
How do you feel about one of these?

```suggestion
    <string name="revanced_replace_player_cast_button_summary">"Replaces the cast button in the player with an 'Open music' button, which opens the official audio instead of the music video. (Experimental)

Info: This feature does not work normally if any songs in the playlist cannot be played."</string>
```

Or alternatively:

```suggestion
    <string name="revanced_replace_player_cast_button_summary">"Replaces the cast button in the player with an 'Open music' button. (Experimental)

Info: 
• The 'Open music' button opens the official audio instead of the music video.
• This feature does not work normally if any songs in the playlist cannot be played."</string>
```